### PR TITLE
First file to create a pull request

### PR DESCRIPTION
**Pull request by BharadwajJannu**
This file tells the adding a new file to my repository 
Pull request to #42

### CODE REVIEW

**Comment by BharadwajJannu** [comment]
Test the comment

**Comment by kg255000** [comment]
Hi Bharadwaa....

File: BJ/newfile.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Create new .md file and commit in my repository
+
+- This file was created in sub directory BJ
+
+
+## Modifications to my file in local repository
+- Update this file
```
Comment (tjshelley) [inline BJ/newfile.md]: Hi My name is tim, are you enjoying the course?

**Comment by tjshelley** [comment]
I have added a comment you your file.

File: BJ/newfile.md (hunk 1 above)
Comment (BharadwajJannu) [inline BJ/newfile.md]: Test the second comment

**Pull request merged**